uint8 id
---
bool success
geometry_msgs/Point[] partition